pyEpiabm/pyEpiabm/utility/covidsim_kernel.py

```diff
@@ -0,0 +1,31 @@
+#
+# Class of Covidsim-style gravity kernels
+#
+
+
+class Kernel:
+    """Class to create the gravity kernel used throughout
+    CovidSim in the spatial weighting.
+    """
+    @staticmethod
+    def weighting(distance: float, scale: float = 4, shape: float = 3.8):
+        """Returns the weighting given by the Covidsim distance
+        weighting. Default parameters use the Covidsim GB spatial data.
```

Comment:
```suggestion
        weighting. Default parameters use the Covidsim GB spatial data.
        
        The formula is given by
        
        .. math::
            \frac{1}{\left(1+\frac{\text{distance}}{\text{scale}}\right)^\text{shape}}
```
While it is easy enough to read from the source code in this case, it doesn't hurt to give it in the docstring, and I'm also curious whether we have the latex working on readthedocs and sphinx.